channels/binding/websockets.py

```diff
@@ -87,3 +87,42 @@ def update(self, pk, data):
         for name in data.keys():
             setattr(instance, name, getattr(hydrated.object, name))
         instance.save()
+
+
+class WebsocketBindingWithMembers(WebsocketBinding):
+    """
+    Outgoing binding subclass based on WebsocketBinding. Additionally
+    enables sending of member variables, properties and methods.
+    Member methods can only have self as a required argument.
+    Just add the name of the member to the send_members-list.
+    Example:
+
+    class MyModel(models.Model):
+        my_field = models.IntegerField(default=0)
+        my_var = 3
+
+        @property
+        def my_property(self):
+            return self.my_var + self.my_field
+
+        def my_function(self):
+            return self.my_var - self.my_vield
+
+    class MyBinding(WebsocketBindingWithMembers):
+        model = MyModel
+        stream = 'mystream'
+
+        send_members = ['my_var', 'my_property', 'my_function']
+    """
+
+    send_members = []
+
+    def serialize_data(self, instance):
+        data = super(WebsocketBindingWithMembers, self).serialize_data(instance)
+        for m in self.send_members:
+            member = getattr(instance, m)
+            try:
+                data[m] = member()
+            except TypeError:
+                data[m] = member
+        return data
```

Comment:
Maybe consider catching/warning about types that won't serialize into JSON; for the fields, this is done automatically with the Django serializer.